### PR TITLE
Fix: Fixed NullReferenceException with DefaultDestinationFolderPath

### PR DESCRIPTION
**Pull request by yaira2**
**Resolved / Related Issues**
- [x] Were these changes approved in an issue or discussion with the project maintainers? In order to prevent extra work, feature requests and changes to the codebase must be approved before the pull request will be reviewed. This prevents extra work for the contributors and maintainers.
   Fixes https://appcenter.ms/orgs/FilesApp/apps/Files/crashes/errors/3137641778u/overview

### CODE REVIEW

**Review by hishitetsu — COMMENTED** [review]
I think we need to consider the case where `SlimContentPage` is null.

**Comment by yaira2** [comment]
In `DecompressArchiveDialogViewModel`?

**Comment by hishitetsu** [comment]
Here, for example.
https://github.com/files-community/Files/blob/f5b10a84b42f92c2fd35a083bc3cb67f18cd7059/src/Files.App/Utils/Archives/DecompressHelper.cs#L211

**Comment by yaira2** [comment]
Do you think that's enough, or should I keep my other changes as well?

**Comment by hishitetsu** [comment]
I don't think the archive will ever be null, but it could be left for safety.

File: src/Files.App/Utils/Archives/DecompressHelper.cs [hunk 1]
```diff
@@ -172,12 +172,18 @@ public static async Task DecompressArchiveHereAsync(IShellPage associatedInstanc
 			if (associatedInstance == null)
 				return;
 
+			if (associatedInstance.SlimContentPage == null)
+				return;
```
Comment (hishitetsu) [inline src/Files.App/Utils/Archives/DecompressHelper.cs:176]:
```suggestion
			if (associatedInstance?.SlimContentPage == null)
				return;
```

File: src/Files.App/Utils/Archives/DecompressHelper.cs [hunk 2]
```diff
@@ -205,6 +211,9 @@ public static async Task DecompressArchiveToChildFolderAsync(IShellPage associat
...
```
Comment (hishitetsu) [inline src/Files.App/Utils/Archives/DecompressHelper.cs:215]:
```suggestion
			if (associatedInstance?.SlimContentPage == null)
				return;
```

File: src/Files.App/Utils/Archives/DecompressHelper.cs (hunk 1 above)
Comment (yaira2) [reply src/Files.App/Utils/Archives/DecompressHelper.cs:176]: This is much better, thanks!

**Pull request merged**